.travis.yml

```diff
@@ -1,13 +1,16 @@
 # Run tests run on travis-ci after each commit and before each merge
 
 language: java
+dist: trusty
```

Comment:
Nope it doesn't work without trusty (see [here](https://travis-ci.org/javydreamercsw/pcgen/builds/240824494)). Please expand on the conflicts you refer to. If we want the installer to be build in Travis there's no other way I'm aware of besides replacing all those Perl scripts used in various parts. Not a task I could take any time soon.

Any feedback @LegacyKing ?